client/packages/common/src/types/schema.ts

```diff
@@ -3786,6 +3786,11 @@ export type StoreNode = {
   __typename: 'StoreNode';
   code: Scalars['String'];
   id: Scalars['String'];
+  /**
+   * Returns the associated store logo.
+   * The logo is returned as a data URL schema, e.g. "data:image/png;base64,..."
+   */
+  logo?: Maybe<Scalars['String']>;
```

Comment:
Not related to this PR, but was auto-generated -- must be related to another recent change.